pom.xml

```diff
@@ -14,8 +14,6 @@
 
   <build>
 
-    <finalName>lti-utils</finalName>
```

Comment:
So the reason I did this was because whenever I compiled the jar via mvn clean install, the final name would just be lti-utils.jar. When I removed it, the name became lti-utils-1.5.jar, which I believe is what we want.